libs/block-editor/src/lib/models/dot-bubble-menu.model.ts

```diff
@@ -23,6 +24,7 @@ export interface BubbleMenuItem {
 export interface BubbleMenuComponentProps {
     items: BubbleMenuItem[];
     command: EventEmitter<BubbleMenuItem>;
+    dropdown: BubbleChangeDropdownComponent;
```

Comment:
Too tight... what happen is we want to add more components here?

Reply:
this is a reflection of the component `@Inputs`, so it will be that component always.